cookbook/ssh_agent.md

```diff
@@ -26,22 +26,29 @@ See the workarounds.
 You can work around this behavior by checking if a ssh-agent is already running on your user, and start one if none is:
 
 ```nushell
-# exemple for env.nu
-let sshAgentFilePath = $"/tmp/ssh-agent-($env.USER).nuon"
-
-if ($sshAgentFilePath | path exists) and ($"/proc/((open $sshAgentFilePath).SSH_AGENT_PID)" | path exists) {
-    # loading it
-    load-env (open $sshAgentFilePath)
-} else {
-    # creating it
-    ^ssh-agent -c
+do --env {
+    let ssh_agent_file = (
+        $nu.temp-path | path join $"ssh-agent-($env.USER? | default $env.USERNAME).nuon"
```

Comment:
you wrote `$env.USER? | default $env.USERNAME` for crossplatform compatibility but `$env.USERNAME` doesn't exists on linux, so i think both `$env` user variables would need a `?` and thus maybe a check